ios/Classes/PdftronFlutterPlugin.h

```diff
@@ -117,6 +123,20 @@ static NSString * const EVENT_ANNOTATION_CHANGED = @"annotation_changed_event";
 static NSString * const EVENT_ANNOTATIONS_SELECTED = @"annotations_selected_event";
 static NSString * const EVENT_FORM_FIELD_VALUE_CHANGED = @"form_field_value_changed_event";
 
+// fit mode
+static NSString * const PTFitPageKey = @"FitPage";
+static NSString * const PTFitWidthKey = @"FitWidth";
+static NSString * const PTFitHeightKey = @"FitHeight";
+static NSString * const PTZoomKey = @"Zoom";
+
+// layout mode
+static NSString * const PTSingleKey = @"Single";
+static NSString * const PTContinuousKey = @"Continuous";
+static NSString * const PTFacingKey = @"Facing";
+static NSString * const PTFacingContinuousKey = @"FacingContinuous";
+static NSString * const PTFacingCoverKey = @"FacingCover";
+static NSString * const PTFacingCoverContinuousKey = @"FacingCoverContinuous";
```

Comment:
The previous string constants do not start with an uppercase letter, but these new ones all do. They should be consistent.